openedx_authz/api/data.py

```diff
@@ -181,6 +181,14 @@ def __call__(cls, *args, **kwargs):
         if cls is not ScopeData:
             return super().__call__(*args, **kwargs)
 
+        # When working with global scopes, we can't determine subclass with an external_key since
+        # a global scope it's not attached to a specific resource type. So we only use * as an
+        # an external_key to mean generic scope which maps to base ScopeData class.
+        # The only remaining issue is that internally the namespace key used in policies will be
+        # The generic scope namespace (sc^*), so we need to handle that case here.
+        if kwargs.get("external_key") == GENERIC_SCOPE_WILDCARD:
```

Comment:
Generic or global?

Reply:
I initially used _generic scope_ cause this scope can be specialized into other scopes: content libraries, courses... But this might be too ambiguous. It might make sense making it global when not specialized?